Yona/Yona/CustomCells/YonaUserTableViewCell.swift

```diff
@@ -116,18 +104,26 @@ class YonaUserTableViewCell: PKSwipeTableViewCell {
         
         let formatter = DateFormatter()
         formatter.dateFormat = "HH:mm"
-        
         if aMessage.messageType == notificationType.GoalConflictMessage {
             boldLineLabel.text = aMessage.simpleDescription() + " " + aMessage.activityTypeName + " - " + formatter.string(from: aMessage.creationTime as Date)
+            avatarImageView.image = R.image.adultSad()
         } else {
             boldLineLabel.text = aMessage.simpleDescription()
+            if let link = aMessage.userPhotoLink,
+                let URL = URL(string: link) {
+                avatarImageView.kf.setImage(with: URL)
+            } else {
+                avatarImageView.image = nil
+            }
         }
+        
         normalLineLabel.text = "\(aMessage.nickname)"
      
         var tmpnickname = ""
         tmpnickname = aMessage.nickname
-        if tmpnickname.characters.count > 0 {
-            avatarNameLabel.text = "\(tmpnickname.capitalized.characters.first!)"
+        
+        if tmpnickname.count > 0 && aMessage.messageType != notificationType.GoalConflictMessage && aMessage.userPhotoLink == nil {
```

Comment:
If ``aMessage.messageType != notificationType.GoalConflictMessage``, then ``aMessage.userPhotoLink == nil`` is never true, so you can simplify this condition:
```
if tmpnickname.count > 0 && aMessage.userPhotoLink == nil {
```

Reply:
True. simplified condition.